### PR TITLE
BZ-1699757: Add oc adm release and example to CLI reference

### PR DESCRIPTION
**Pull request by bergerhoffer**
@openshift/team-documentation For peer review please.

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1699757/cli_reference/administrator-cli-commands.html#release

### CODE REVIEW

File: modules/cli-administrator-other.adoc [hunk 1]
```diff
@@ -38,6 +38,18 @@ $ oc adm config view
 $ oc adm config use-context test-context
 ----
 
+== release
+
+Manage various aspects of the OpenShift release process, such as viewing information about a release or inspecting the contents of a release.
```
Comment (ahardin-rh) [inline modules/cli-administrator-other.adoc]: s/OpenShift/{product-title}

**Comment by ahardin-rh** [comment]
@bergerhoffer Just one minor suggestion from me. Looks good! :+1:

File: modules/cli-administrator-other.adoc [hunk 2]
```diff
@@ -38,6 +38,18 @@ $ oc adm config view
...
+
+.Example: Generate a changelog between two releases and save to `changelog.md`
+----
+$ oc adm release info --changelog=/tmp/git \
+    quay.io/openshift-release-dev/ocp-release:4.0.0-0.9 \
+    quay.io/openshift-release-dev/ocp-release:4.0.0-0.11 \
+    > changelog.md
```
Comment (sferich888) [inline modules/cli-administrator-other.adoc]: /LGTM

@smarterclayton and @tbielawa you may want to review

**Comment by xingxingxia** [comment]
LGTM, thanks

**Pull request merged**